fili-system-config/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>spock-core</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.spockframework</groupId>
+            <artifactId>spock-core</artifactId>
+            <scope>compile</scope>
```

Comment:
This appears to be a mistake. At best, spock-core should be test scope

Reply:
Addressed. Thanks for pointing it out.